controllers/provider-alicloud/charts/internal/seed-controlplane/charts/alicloud-cloud-controller-manager/templates/configmap-monitoring.yaml

```diff
@@ -48,4 +47,5 @@ data:
             summary: Cloud controller manager is down.
 
   dashboard_users: |
```

Comment:
Please add the cloud-controller-manager dashboard to the `dashboard_operators`, too.